tools/EMLassemblyline/make_eml.xml

```diff
@@ -0,0 +1,67 @@
+<tool id="makeeml" name="Make EML" version="0.1.0+galaxy0">
+    <description> Create EML from EAL templates</description>
+    <requirements>
+        <requirement type="package" version="4.3.1">r-base</requirement>
+        <requirement type="package" version="3.5.5">r-emlassemblyline</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    mkdir output_template &&
+    cd output_template &&
+    #for $input in $templates
+        cp '$input' '${input.element_identifier}.txt' &&
+    #end for
+    find . -type f -empty -print -delete &&

```

Comment:
why is this needed? Can you not check that inside the R script?

Reply:
Not so important I think. I remove it and if needed, we can manage this differently